pkg/pillar/scripts/onboot.sh

```diff
@@ -148,7 +148,7 @@ free_space() {
 }
 
 # Checking for low diskspace at bootup.
-# If there is less than 4Mbytes (MIN_DISKSPACE) then remove the content of the
+# If there is less than 4096 Mbytes (MIN_DISKSPACE) then remove the content of the
```

Comment:
the comment in the beginning of the file is wrong, `free_space()` function uses `df -k` which will give the size in KBytes, so 4096 KB is indeed  4MB, the comment here was correct...